Symphony/Composer.swift

```diff
@@ -22,36 +22,43 @@ public protocol Composer: class {
     /// A reference to the currently presented composable.
     /// This is generally only necissary for memory management,
     /// and shouldn't need to be messed with.
-    var currentComposable: Composable? { get set }
```

Comment:
So, I realized after I started trekking down this path that it's not as obvious a win as I was hoping. While I do think `Set` is the most appropriate data structure (`currentComposables` _should_ be unique and unordered), making `Composable: Equatable & Hashable` is non trivial (especially in Swift 2), so I'm voting we punt for the time being 😞